router/batchrouter/batchrouter.go

```diff
@@ -693,7 +693,16 @@ func (brt *HandleT) copyJobsToStorage(provider string, batchJobs *BatchJobsT, ma
 		})
 		opID = brt.jobsDB.JournalMarkStart(jobsdb.RawDataDestUploadOperation, opPayload)
 	}
+
+	startTime := time.Now()
```

Comment:
@chrikar @mcchran this is a good example why the current stat collection API we are using is too obtrusive. Measuring an operation requires 3 steps, whereas the actual operation that we measure, which is the critical part of the execution, requires only 1.

@lvrach @fracasula

Reply:
What I currently have in mind is something like this
```go
stats.Timed("brt_upload_time", map[string]string{"tag","value"}, func(){
  // do operation here that needs to be measured
})
```
As always, open to better ideas!